pyfar/constants/constants.py

```diff
@@ -433,3 +434,149 @@ def density_of_air(
+    #
+    if num_fractions == 1:
+            if (f_lims[0] > 15.8*G**(-1/2)) and ( f_lims[1] <
+                                                 15848.93192*G**(1/2)):
+                nominal = np.array([
+                    16, 31.5, 63, 125, 250, 500, 1e3,
+                    2e3, 4e3, 8e3, 16e3], dtype=float)
+            else:
+                warnings.warn(
+                "The nominal center frequencies for one-third-octave-band " \
+                "are defined only from 11.2 Hz to 22387.2 Hz.",  stacklevel=2)
+    elif num_fractions == 3:
+        if (f_lims[0] > 10*G**(-1/6)) and (f_lims[1] < 19952.62315*G**(1/6)):
+            nominal = np.array([
+                10, 12.5, 16, 20, 25, 31.5, 40, 50, 63, 80,
+                100, 125, 160, 200, 250, 315, 400, 500, 630,
+                800, 1000, 1250, 1600, 2000, 2500, 3150, 4000,
+                5000, 6300, 8000, 10000, 12500, 16000, 20000], dtype=float)
+        else:
+            warnings.warn(
+                "The nominal center frequencies for octave-band filter are " \
+                "defined only from 8.91 Hz to 22387.2 Hz.",  stacklevel=2)
+
+    mask = (nominal >= f_lims[0]) & (nominal <= f_lims[1])
+    nominal = nominal[mask]
+    return nominal
+
+def fractional_octave_frequencies_exact(
+        num_fractions=1, frequency_range=(20, 20e3)):
+    r"""Return the exact center frequencies for fractional-octave-band filters.
+
+    The frequencies are calculated in accordance with the IEC 61260-1:2014
+    standard [#]_.
+
+    The octave frequency ratio, :math:`G`, is given by the following
+    expression.
+
+    .. math::
+
+        G = 10^{3/10} \approx 1.99526
+
+    The center frequencies :math:`f_m` are calculated using formula
+    :eq:`eq_center_odd` for odd values of :math:`b` and formula
+    :eq:`eq_center_even` for even values of :math:`b`.
+
+    .. math::
+        :label: eq_center_odd
+
+        f_m = f_r \cdot G^{x/b}
+
+    .. math::
+        :label: eq_center_even
+
+        f_m = f_r \cdot G^{(2x+1)/(2b)}
+
+    where:
+
+    - :math:`b` is the number of octave fractions.
+    - :math:`f_r` is the reference frequency, set to 1000 Hz.
+    - :math:`x` is the index of the frequency band.
+
+    Parameters
+    ----------
+    num_fractions : int, optional
+        The number of bands an octave is divided into. Eg., ``1`` refers to
+        octave bands and ``3`` to third octave bands. The default is ``1``.
+        All positive integers are allowed.
+    frequency_range : array, tuple
+        The lower and upper frequency limits, the default is
+        ``(20, 20e3)``.
+
+    Returns
+    -------
+    exact : array, float
+        The exact center frequencies in Hz, resulting in a uniform distribution
+        of frequency bands over the frequency range.
+    cutoff : tuple, array, float
+        The lower and upper cutoff frequencies in Hz of the bandpass filters
+        for each band as a tuple corresponding to ``(f_lower, f_upper)``.
```

Comment:
sorry for messing everthing up again. But would it make sence to return simply 
center
cutoff_lower
cutoff_upper
might be clearer as before. and we wouldnt have an ugly tuple in the cutoff
what dou you guys think? Maybe we can disucss on friday